### PR TITLE
Add script to create new etherpad release

### PR DESCRIPTION
**Pull request by Gared**
Do not merge yet.
This script throws an error somehow on the end of the curl request:
cURL error:23 - Failed writing body 

This error never happens in the function check_api_token, but in merge_release_branch and merge_web_branch. The exception seems to be thrown after the curl requests finished, but I don't know why.

### CODE REVIEW

File: bin/createRelease.sh [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+echo "- temp dir: $TMP_DIR"
+
+# get new version number
+# format: x.x.x
+echo -n "Enter new version (x.x.x): "
+read VERSION
+
+# get the message for the changelogs
+echo -n "Enter new changelog entries (enter a blank line to end): "
+while read -r line; do
```
Comment (0ip) [inline bin/createRelease.sh]: I could imagine, something like this:
```
read -p "Enter new changelog entries (enter a blank line to end):"
tmp=$(mktemp)
"${EDITOR:-vi}" $tmp
changelogText=$(<$tmp)
rm $tmp
```
might be more convenient and fault tolerant when typing. :)

**Comment by 0ip** [comment]
You could maybe dump the CURL_RESPONSE variable and check for encoding issues, that may cause premature closing of the pipe.

**Comment by JohnMcLear** [comment]
@Gared bump?

**Comment by Gared** [comment]
@0ip Thanks for your tips. Will check if it works on our next release

**Comment by JohnMcLear** [comment]
@Gared @0ip  bump

**Comment by Gared** [comment]
Script is still throwing this error if you're executing it. Any help on this would be really appreciated

**Comment by muxator** [comment]
Closing this, as `createRelease.sh` was included in 6dc808ad5483 anyway.

@Gared, if you think it can be improved, every contribution is appreciated.

**Pull request closed without merging**